docs/kratos/hooks/01_configure-hooks.mdx

```diff
@@ -0,0 +1,373 @@
+- `after` verification: The hook runs when the user verifies their account.
+
+### Hooks for specific methods
+
+The login and registration self-service flows support multiple authentication methods:
+
+| Authentication method | Description                                                                                    |
+| :-------------------- | :--------------------------------------------------------------------------------------------- |
+| `password`            | Sign-in and sign-up with username and password combo.                                          |
+| `oidc`                | Sign-in and sign-up through OIDC-compliant OAuth2 identity providers.                          |
+| `totp`                | Sign-in with a TOTP code from apps such as Google Authenticator.                               |
+| `webauthn`            | Sign-in with WebAuthn-compatible factors (FaceID, YubiKey) or paswordless sign-up and sign-in. |
+| `lookup_secret`       | Sign-in with recovery codes.                                                                   |
+
+:::tip
+
+You can use the authentication methods to further specify when you want to run hooks. For example, you can run a hook for the
+login flow with the `password` method and not run any hooks for logins with the `oidc` method.
+
+:::
+
+## Available hooks
+
+There are 4 hooks you can use to extend self-service user flows. Note that some hooks can only be used for specific flows and
+methods.
+
+| Hook                       | Description                                                                                       | Constraints                                                                                              |
+| :------------------------- | :------------------------------------------------------------------------------------------------ | :------------------------------------------------------------------------------------------------------- |
+| `session`                  | Signs in the user immediately after they create an account.                                       | For use only with the `after` registration flow. To use it, you must define secrets for secret rotation. |
+| `revoke_active_sessions`   | Revokes all other active sessions of the user on successful login.                                | For use only with the login flow.                                                                        |
+| `require_verified_address` | Allows users to sign in only when they verified their email address.                              | For use only with the `after` login flow, `password` method only.                                        |
+| `web_hook`                 | Allows to trigger external logic. Can be used with all flows and methods except error and logout. | Requires providing web-hook configuration. The only hook type available for the `after` settings flow.   |
+
```

Comment:
I think something is unclear in this part:

We don't explicitly say that if we do not specify a AuthN method, the hook will apply to all flows.